app/views/users/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="user_show container">
+
+  <div class="user_item row">
+    <div class="u_img">
+      <img src=<%= @user.photo %> alt="User Image">
+    </div>
+
+    <div class="user_cont">
+      <a href=<%= user_url(@user.id) %>>
+        <div class="user_desc col">
+          <h2 class="user_name"><%= @user.name %></h2>
+          <p class="user_post_n">Number of posts: <%= @user.posts_count %></p>  
+        </div>
+      </a>
+    </div> 
+  </div>
+
+  <div class="user_bio">
+    <p class="show_title">Bio</p>
+    <p class="bio_detail"><%= @user.bio %></p>
+  </div>
+
+  <ul class="post_list col">
+    <% @recent_posts.each do |post| %>
```

Comment:
[REQUIRED] I can see a `@recent_posts` method in the show action present in your users controller. Also, please remember that you are expected to use the recent_posts method you created in your user model to fetch the recent_posts. Kindly change your code to use the recent_post method.